src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -229,6 +230,13 @@
     ScoreboardBuilder getScoreboardBuilder();
 
     /**
+     * Gets the main {@link Scoreboard} controlled by the server.
+     *
+     * @return The main {@link Scoreboard} controlled by the server
+     */
+    Scoreboard getMainScoreboard();
```

Comment:
@simon816 I wasn't entirely sure of where to put it so that may be the case, but I could also see an argument being made for `GameRegistry`.

@modwizcode The main scoreboard is the one attached to the /scoreboard command.